index.html

```diff
@@ -23,79 +23,149 @@
+      </p>
       <div class="features">
-
-        <span class="fa fa-rocket fa"></span><br>
-        <h3>Interactive Queries</h3>
-        <p>
-          Issue sub-second ad-hoc queries to group, filter, and aggregate data. Druid is ideal for powering multi-tenant user-facing applications. 
-        </p>
-
-        <span class="fa fa-feed fa"></span><br>
-        <h3>Real-time Streams</h3>
-        <p>
-          Explore events immediately after they occur. Ingest data in streams or batches to unify real-time and historical views.
-        </p>
-
-        <span class="fa fa-database fa"></span><br>
-        <h3>Horizontally Scalable</h3>
-        <p>
-          Existing Druid clusters have scaled to petabytes of data and
-          trillions of events, ingesting millions of events every second. Druid is
-          extremely cost effective, even at scale. 
-        </p>
-
-        <span class="fa fa-gears fa"></span><br>
-        <h3>Deploy Anywhere</h3>
-        <p>
-          Druid runs on commodity hardware. Deploy it in the cloud or
-          on-premise. Integrate with existing big data systems such as
-          Hadoop, Spark, Kafka, Storm, Flink, and Samza.
-        </p>
-
-        <span class="fa fa-comments fa"></span><br>
-        <h3>Vibrant Community</h3>
-        <p>
-          Druid is a community led project. <a href="/community">Join the fast growing community</a> and
-          work with developers from across the world.
-        </p>
-
+        <div class="feature">
+          <span class="fa fa-columns fa"></span>
+          <h5>Column-oriented storage</h5>
+          <p>
+            Druid stores and compresses each column individually, and only needs to read the ones needed for a particular query, which supports fast scans, rankings, and groupBys.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-search fa"></span>
+          <h5>Native search indexes</h5>
+          <p>
+            Druid creates inverted indexes for string values for fast search and filter.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-tint fa"></span>
+          <h5>Streaming and batch ingest</h5>
+          <p>
+            Out-of-the-box connectors for Apache Kafka, HDFS, AWS S3, stream processors, and more.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-stream fa"></span>
+          <h5>Flexible schemas</h5>
+          <p>
+            Druid gracefully handles evolving schemas and <a href="/docs/latest/ingestion/flatten-json">nested data</a>.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-clock fa"></span>
+          <h5>Time-optimized partitioning</h5>
+          <p>
+            Druid intelligently partitions data based on time and time-based queries are significantly faster than traditional databases.
```

Comment:
What do you think about removing "time optimized partitioning" (it's cool but I'm not sure it's top 7 features) and adding instead this one:

- *Easy to operate.* Scale up or down by just adding or removing servers, and Druid automatically rebalances. Fault-tolerant architecture routes around server failures.

And link "fault tolerant architecture" to the architecting for failure blog post.

Reply:
I believe time optimized partitioning is important

I think this bullet point should be repeated on the technology page